java/ql/src/semmle/code/java/frameworks/android/SharedPreferences.qll

```diff
@@ -0,0 +1,51 @@
+import java
```

Comment:
We still need the top-level qldoc for the file itself (this is called file-module qldoc).
```suggestion
/** Provides classes related to `android.content.SharedPreferences`. */

import java
```

Reply:
Changed